### ISSUE TITLE
Icons on Settings window missing when using an external theme

### ISSUE THREAD

**Issue by Scrxtchy** [pathmann/pyTSon#48]
I installed [DarkenTS](https://www.myteamspeak.com/addons/0b57d54d-b46c-433d-8f7e-2eea28470007) today,
When opening the settings window, reload.png and delete.png are not loaded
![](https://jii.moe/rJHEfBNte.png)
Compared to [this image](https://jii.moe/HyA-mBEKl.png) where I have loaded the `default_coloured_2014` icon pack. Which are stored as zip files within the application install directory, compared to that of the user directory for addons

I manually added a `delete.png` as it the icon pack was missing it thinking it could have been an issue, but even with the new file, it did not fix the issue. `reload.png` existed, but was still effected

The following is spat into the client log
```
17/02/2017 09:28:05	pytsonui.retrieveWidgets.	Error	Error loading iconpack: 'str' object has no attribute 'decode'	
17/02/2017 09:28:05	pyTSon.ConfigurationDialog.setupList	Error	Error loading iconpack: 'str' object has no attribute 'decode'	
```

**Comment by Bluscream** [comment]
I was about to report this too, but i thought @pathmann might be too busy

**Comment by Scrxtchy** [comment]
It doesn't really affect functionality. no hassle to get it done as it's just an aesthetic error

**Comment by pathmann** [comment]
Fixed with 464935b.
